examples/neo4j/README.md

```diff
@@ -0,0 +1,28 @@
+# Neo4j example
+
+A simple app using tinyhttp and [neo4j](https://neo4j.com/docs/).
+
+## Setup
+
+```sh
+tinyhttp new neo4j
+```
+
+## Run
```

Comment:
please run prettier on the readme as well, there's no space between a heading and a paragraph